src/frontend/src/handler/create_table.rs

```diff
@@ -363,11 +379,11 @@ pub fn bind_sql_table_column_constraints(
         })
         .try_collect()?;
 
-    let mut columns_catalog = columns_descs
+    let mut columns_catalog: Vec<ColumnCatalog> = columns_descs
         .into_iter()
         .map(|c| {
             // All columns except `_row_id` or starts with `_rw` should be visible.
-            let is_hidden = c.name.starts_with("_rw");
+            let is_hidden: bool = c.name.starts_with("_rw");
```

Comment:
Just curious why is this type annotation added

Reply:
I guess it is produced when copying and pasting the code. thanks!